01/co2_annmean_mlo.txt

```diff
@@ -0,0 +1,110 @@
+# --------------------------------------------------------------------
```

Comment:
Please add a read me to this directory to document where you got this data from.